deeppavlov/models/embedders/fasttext_embedder.py

```diff
@@ -50,14 +50,8 @@ def load(self, *args, **kwargs):
             log.info("[loading embeddings from `{}`]".format(self.load_path))
             model_file = str(self.load_path)
             if self.emb_module == 'fasttext':
-                import fasttext as Fasttext
+                import fastText as Fasttext
                 model = Fasttext.load_model(model_file)
```

Comment:
Can we throw this if out and do the import at the top of the file?

Reply:
Done.